### PR TITLE
 Add bedrock.externalpages app for Pocket marketing pages #10645 (recovered)

### PR DESCRIPTION
**Pull request by alexgibson**
## Description

This PR adds a new Django app called "externalpages" to house the Pocket marketing pages that will be proxied to bedrock under the URL `/external/pocket/`.

The first several pages have been moved from pocket-marking-pages into bedrock:
 - add
 - about
 - chrome
 - android
 - opera
 - ios
 - edge

The front-end of this app is mostly ported from pocket-marketing-changes, so there is some code that could be improved, but we focused on moving things over and figured much of the improvements would come from the redesign next year.

## Issue
#10211

### Front end
I would test http://localhost:8000/externalpages/pocket/about and http://localhost:8000/externalpages/pocket/add against those game pages from www.getpocket.com/add www.getpocket.com/about

### CODE REVIEW

**Review by maureenlholland — COMMENTED** [review]
The overall structure looks good to me! I have a list of nitty styling things that can wait for a clean up PR. 

Minor clean-up requested here but nothing blocking.

File: bedrock/externalpages/templates/externalpages/pocket/about.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+{% endblock  %}
+
+{% block content %}
+{% include 'externalpages/pocket/includes/nav.html' %}
+
+    <div class="intro-wrapper">
+        <section class="about-section first">
+            <div class="about-section-text-wrapper">
+                <h1 class="about-section-heading">Hi, we're Pocket.</h1>
+                <p class="about-section-body"> We’re powered by a community of interesting people with curious minds, and we empower people to keep their minds focused, stimulated, and nourished. Which makes us a refreshing change fromother digital spaces.</p>
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/externalpages/templates/externalpages/pocket/about.html:22]: Good eye!!

File: bedrock/externalpages/templates/externalpages/pocket/platform-pages.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block page_title %} Add Pocket to Chrome {% endblock %}
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/externalpages/templates/externalpages/pocket/platform-pages.html:11]: Good idea! I changed it to have `self.page_title()`

**Review by maureenlholland — APPROVED** [review]
r+ ✅

**Pull request merged**